### PR TITLE
i18n(es): add `hashnode.mdx` 

### PR DESCRIPTION
**Pull request by ItzRoBeerT**
#### Description (required)

Add new `hashnode.mdx` translation for `es`.
Reused the original `hashnode.mdx` file, from `en` in order to translate the new file.

#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/cms/hashnode.mdx [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+  yarn add graphql-request
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## Haciendo un blog con Astro y Hashnode
+
+Esta guía usa [`graphql-request`](https://github.com/jasonkuhrt/graphql-request), un cliente GraphQL minimalista que funciona bien con Astro, para traer tus datos de Hashnode a tu proyecto de Astro.
+
+### Requisitos Previos
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/hashnode.mdx:52]:
```suggestion
### Prerrequisitos
```

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @ItzRoBeerT, I left you just a small suggestion for consistency.

File: src/content/docs/es/guides/cms/hashnode.mdx (hunk 1 above)
Comment (ItzRoBeerT) [reply src/content/docs/es/guides/cms/hashnode.mdx:52]: Thanks for the feedback @dreyfus92!! 
You're right!! i should change that!

**Comment by dreyfus92** [comment]
@ItzRoBeerT I haven't merged this yet cause the change hasn't been applied 😁

File: src/content/docs/es/guides/cms/hashnode.mdx [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+
+[Hashnode](https://hashnode.com/) es un CMS alojado que te permite crear un blog o publicación.
+
+## Integración con Astro
+La [API Pública de hashnode](https://apidocs.hashnode.com/) es una API GraphQL que te permite interactuar con Hashnode. Esta guía utiliza [`graphql-request`](https://github.com/jasonkuhrt/graphql-request), un cliente GraphQL minimalista que funciona bien con Astro, para traer tus datos de Hashnode a tu proyecto de Astro.
+
+### Prerrequisitos
+Para comenzar necesitarás tener lo siguiente:
+
+1. **Un proyecto de Astro** - Si aún no tienes un proyecto de Astro, nuestra [Guía de Instalación](/es/install/auto/)te ayudará a empezar en poco tiempo.
```
Comment (yanthomasdev) [inline src/content/docs/es/guides/cms/hashnode.mdx:22]:
```suggestion
1. **Un proyecto de Astro** - Si aún no tienes un proyecto de Astro, nuestra [Guía de Instalación](/es/install-and-setup/) te ayudará a empezar en poco tiempo.
```

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work, y'all. @ItzRoBeerT, I saw the suggestion you agreed with by didn't apply, I hope you don't mind I took the liberty to apply it for you!

**Pull request merged**